packages/main/src/plugin/kubernetes/contexts-informers-registry.ts

```diff
@@ -36,6 +36,9 @@ export class ContextsInformersRegistry {
   }
 
   setInformers(name: string, informers: ContextInternalState | undefined): void {
+    if (this.informers.get(name)) {
+      console.debug(`primary informers in context ${name} already set`);
```

Comment:
Are we gonna keep this trace ? Shouldn't we return in that case ?

Reply:
Yes, I would like to keep the trace, and not return here.

The purpose is to check that the informer was correctly deleted from the map before we set it again here, to avoid any memory leak. In any case, we want to set the new one.

Reply:
if there is something in the map, shouldn't we do something about it instead of just logging a trace

I don't see how the trace will be useful 

Either there is a bug and it needs to be fixed or there are tests

Reply:
Yes, I completely get your point and I also had the same thought. 

About fixing the bug, I still have work to do related to this issue and others (in other PRs), these traces will help me detect if there are bugs when I work on these PRs.

About tests, it is not straightforward to cover all the possibilities, as the Kubernetes API and the Kubernetes library have behaviours we can hardly reproduce completely with unit tests. I would prefer to have this trace (in addition to existing tests) to detect unknown cases or regressions.

Reply:
I still believe this kind of traces is not the way to go

why ? because Podman Desktop is not only a blackbox that we run in a batch, then collect traces and then diagnose them.

We had issues in stores, etc, in the past, so how we did monitor the right usage, by reporting things directly in a way that from UI you can inspect/interact and monitor the status.

so if you want to check "memory leak" then, some data needs to be exposed so we can query it from a page and display the current data, reset the values, etc

Reply:
an example:

I'm flooded by traces about informers backoff when I open the console, while probably if I want to check the state I just want to go to a page where I see informers object, the last time where they were refreshed, I could ask for a refresh, I could ask to delete it, etc.